rfcs/text/0000-image-base-transcriptomics.md

```diff
@@ -0,0 +1,497 @@
+transcriptomics experiments. The following assumptions are made:
+
+1.  Support for an image-based transcriptomics assay in the early
+    iteration phase should be added to the DCP
+
+2.  The current state of the starfish ecosystem is sufficient to
+    fulfill this need and will be selected by the AWG.
+
+3.  This instantiation will require limited scale (low hundreds of
+    datasets over several years)
+
+## **Author(s)**
+
+[Ambrose Carr](mailto:acarr@chanzuckerberg.com)
+
+[Josh Moore](mailto:j.a.moore@dundee.ac.uk)
+
+## **Shepherd**
+[Brian Raymor](mailto:braymor@chanzuckerberg.com)
+
+## **Motivation**
+
+Cellular imaging assays typically either (1) categorize cells based on
+the presence/absence of a particular set of markers, or (2) count
+structures within cells. These assays exist at variable levels of
+magnifications, including the counting of
+[<span class="underline">individual
+atoms</span>](https://www.sciencemag.org/news/2018/07/tour-de-force-researchers-image-entire-fly-brain-minute-detail)
+(Electron Microscopy (EM))
+[<span class="underline">synapses</span>](https://www.hhmi.org/news/how-to-rapidly-image-entire-brains-at-nanoscale-resolution)
+(lattice light sheet), or [<span class="underline">counting molecules
+within cells</span>](http://linnarssonlab.org/osmFISH/) (image-based
+single-cell transcriptomics and image-based proteomics).
+
+Image-based transcriptomics assays are a good candidate for DCP imaging
+support. They are relatively simple imaging assays that transform image
+features into a series of detected transcripts, which are aggregated by
+cell. Because these assays require thin-sections of tissue, they produce
+datasets that are relatively small by imaging standards (1-10s TBs). In
+addition, image-based transcriptomics pipelines generate the same cell x
+gene expression matrix outputted by scRNA-seq assays with additional
+cellular annotations of the location of the cells. Because these data
+structures are already present in the DCP, there are fewer components
+that need to be generalized than for other imaging assays. Finally,
+engineers from
+[*<span class="underline">starfish</span>*](https://spacetx-starfish.readthedocs.io/en/latest/)
+and
+[<span class="underline">OME</span>](https://www.openmicroscopy.org/)
+are working together to craft data formats, validation tooling, and
+processing pipelines that comply with DCP requirements where possible.
+These characteristics make this assay class a good choice for
+prototyping image support in the DCP as it presents the smallest scale
+challenge, the majority of the major deliverables can be provided by
+DCP-extrinsic developers, and it requires generalization of the fewest
+DCP components. Implementation of support for these assays is likely to
+generalize to additional areas of interest to the HCA community such as
+multiplexed antibody imaging and thick-section volumetric imaging for
+both RNA and protein.
```

Comment:
We have imaging based protein datasets ready now and CODEX planned from a few labs. We will need an explicit plan for protein. Imaging based transcriptomics is indeed the start of this support but can we add any specific deliverables related to this? Or add some timelines/plans that wranglers can pass on to contributors?

Reply:
Starfish's roadmap does not include proteomics assays at this time. Our _format_ supports the storage of those data, and starfish would welcome the contribution of a pipeline. However, proteomics is not in-scope for this RFC. 

> 


> We have imaging based protein datasets ready now [...] can we add any specific deliverables related to this? Or add some timelines/plans that wranglers can pass on to contributors?

Could you start an email thread about image-based proteomics? I don't want to expand the scope of this RFC, but agree this is an important question, am interested in what data you have, and want to consider soliciting one or more pipelines to process those data.